checka11y.css

```diff
@@ -87,3 +87,138 @@ a[target = "_blank"]::after /* Links that open a new page/tab target="_blank" */
   content: " WARNING: Links that open in a new tab or window should be communicated to the user, ensure the user knows of this behavior" !important;
   font-family: var(--checka11y-font, cursive);
 }
+
+a > a {
+  border: var(--checka11y-error-border-width, 5px) solid var(--checka11y-error-color, red) !important;
+}
+
+a > a::after {
+  content: "ERROR: Ensure that <a> is not a child of <a>" !important;
```

Comment:
I like these messages, but can we highlight more that they are invalid HTML? So could we fit the phrase "invalid HTML" into all the messages somewhere?

Reply:
`ERROR: Ensure that <a> is not a child of <a> as it as an invalid HTML`
Does this look good?